CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## v 0.5.0 / 2017-11-14
+
+NOTE: This release breaks backward compatibility. `statsd_exporter` now uses
+a YAML configuration file. You must convert your mappings configuration to
+the new format when you upgrade.
+
+* [CHANGE] Use YAML for configuration file
+* [BUGFIX] Fix matching empty statsd metric components #105
+* [CHANGE] Reduce log levels #92
+* [CHANGE] Removed `-statsd.add-suffix` option flag #99
```

Comment:
this is also a breaking change. what's the remedy for users who were relying on it?

Reply:
I can comment that users should remove this flag when they upgrade. They may also have to adjust any graphs, etc.

Reply:
sounds good!